cmd/disk-usage.go

```diff
@@ -18,12 +18,55 @@ package cmd
 
 import (
 	"context"
+	"encoding/json"
+	"fmt"
 	"time"
 )
 
-const (
-	usageCheckInterval = 12 * time.Hour // 12 hours
-)
+// Captures configurable parameters of usage check.
+type usageConfig struct {
+	UsageCheckInterval time.Duration
+}
+
+// MarshalJSON - encodes to JSON data.
+func (u usageConfig) MarshalJSON() ([]byte, error) {
+	type _usageConfig struct {
+		UsageCheckInterval string `json:"interval"`
+	}
+	return json.Marshal(_usageConfig{u.UsageCheckInterval.String()})
+}
+
+// parseDuration - parse duration string
+func parseDuration(dStr string) (time.Duration, error) {
+	d, err := time.ParseDuration(dStr)
+	if err != nil {
+		return d, err
+	}
+	if d < globalMinimumUsageCheckInterval {
+		return d, fmt.Errorf("interval %s is not allowed, minimum required value is %s",
+			d, globalMinimumUsageCheckInterval)
```

Comment:
this check doesn't seem to be executed.pls check
```
➜  minio git:(09a1d039) export MINIO_USAGE_CHECK_INTERVAL=1h45m
➜  minio git:(09a1d039) ./minio server ~/export                
ERROR Unable to parse MINIO_USAGE_CHECK_INTERVAL value (`1h45m`): Invalid usage check interva
      l value.
      > Please check the passed value.
      HELP:
        MINIO_USAGE_CHECK_INTERVAL: Valid usage check interval duration string is a signed sequence of
       decimal numbers,
      each with optional fraction and a unit suffix, such as "2h45m".
      Valid time units are "ns", "us", "ms", "s", "m", "h".
```

Reply:
Check is executed the message is only printed later.